tools/accuracy_checker/accuracy_checker/adapters/__init__.py

```diff
@@ -52,7 +52,8 @@
     FaceDetectionAdapter,
     FaceDetectionRefinementAdapter,
     FasterRCNNONNX,
-    TwoStageDetector
+    TwoStageDetector,
+    DETRAdapter
```

Comment:
Maybe we need to add DETRAdapter to __all__ list

Reply:
already fixed